src/workers/parser/mapExpression.js

```diff
@@ -7,6 +7,22 @@
 import mapOriginalExpression from "./mapOriginalExpression";
 import mapExpressionBindings from "./mapBindings";
 
+import { hasSyntaxError } from "./validate";
+import { buildScopeList } from "./getScopes";
+import generate from "@babel/generator";
+import * as t from "@babel/types";
+
+function handleTopLevelAwait(expression) {
+  if (hasSyntaxError(expression) && expression.match(/await/)) {
+    const newExpression = `(async () => { return ${expression} })().then(r => console.log(r))`;
```

Comment:
This could be improved...

We likely would want to do the transform on the AST level

1. create an async function
2. add the first n-1 statements
3. add a return statement with the value of nth statement.